### ISSUE TITLE
Add LWJGL3 as a dependency for gradle

### ISSUE THREAD

**Issue by eternalcode0** [sriharshachilakapati/SilenceEngine#64]
LWJGL3 has maven artifacts so why not add it to gradle's dependency management?

```groovy
repositories {
    mavenCentral()
    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
}

project.ext.lwjglVersion = "3.0.0-SNAPSHOT"

dependencies {
    compile "org.lwjgl:lwjgl:${lwjglVersion}"
    compile "org.lwjgl:lwjgl-platform:${lwjglVersion}:natives-windows"
    compile "org.lwjgl:lwjgl-platform:${lwjglVersion}:natives-linux"
    compile "org.lwjgl:lwjgl-platform:${lwjglVersion}:natives-osx"
}
```
^^^ taken from [Badlogic's lwjgl-maven-gradle](https://github.com/badlogic/lwjgl3-maven-gradle) project

**Comment by sriharshachilakapati** [comment]
This is a good suggestion, will add this in when separating the backends from the core engine.

**Comment by sriharshachilakapati** [comment]
This has been implemented in the 1.0.1 branch, and the changes are already implemented into the project creator tool. Thanks @eternalcode0 for the suggestion.
